MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -61,6 +61,9 @@ public partial class AudioFileProber
         [GeneratedRegex(@"I:\s+(.*?)\s+LUFS")]
         private static partial Regex LUFSRegex();
 
+        [GeneratedRegex(@"REPLAYGAIN_TRACK_GAIN:\s+(.*?)\s+dB")]
```

Comment:
```suggestion
        [GeneratedRegex(@"REPLAYGAIN_TRACK_GAIN:\s+([0-9.]+)\s+dB")]
```

Would this regex have the same desired group? I prefer having the group be specific in what it's trying to capture

Reply:
Added "REPLAYGAIN_TRACK_GAIN:\s+-?([0-9.]+)\s+dB" to latest push. This allows negative gains too.